src/query/storages/parquet/src/parquet_part.rs

```diff
@@ -30,35 +29,37 @@ use crate::parquet_rs::ParquetRSRowGroupPart;
 
 #[derive(serde::Serialize, serde::Deserialize, PartialEq, Eq, Debug, Clone)]
 pub enum ParquetPart {
-    ParquetFiles(ParquetFilesPart),
+    ParquetFile(ParquetFilePart),
     ParquetRSRowGroup(ParquetRSRowGroupPart),
 }
 
 impl ParquetPart {
     pub fn uncompressed_size(&self) -> u64 {
         match self {
-            ParquetPart::ParquetFiles(p) => p.uncompressed_size(),
+            ParquetPart::ParquetFile(p) => p.uncompressed_size(),
             ParquetPart::ParquetRSRowGroup(p) => p.uncompressed_size(),
         }
     }
 
     pub fn compressed_size(&self) -> u64 {
         match self {
-            ParquetPart::ParquetFiles(p) => p.compressed_size(),
+            ParquetPart::ParquetFile(p) => p.compressed_size(),
             ParquetPart::ParquetRSRowGroup(p) => p.compressed_size(),
         }
     }
 }
 
 #[derive(serde::Serialize, serde::Deserialize, PartialEq, Eq, Debug, Clone)]
-pub struct ParquetFilesPart {
```

Comment:
used to grouping small files for 2 reasons:

1. read multi small files in parallel, just like read multi "column group" of a group in parallel
2. avoid unbalance in cluster, otherwise,  shuffle may help but can not avoid bad cases,  weighted partition is better?

Reply:
Small files will be read by `RangeMerger` now.

`weighted partition` is good idea, it's useful for parquet stage table,  maybe we can address it in another pr?